.github/workflows/automated-release.yml

```
@@ -0,0 +1,84 @@
+      - name: Install standard-version
+        run: npm install -g standard-version
+
+      - name: Bump version
+        env:
+          PAT_TOKEN: ${{ secrets.BOT_ACTIONS_ACCESS_TOKEN }}
+        run: |
+          standard-version --release-as ${{ env.level }}
+          git push --follow-tags "https://${PAT_TOKEN}@github.com/0xSpaceShard/starknet-rivet.git" HEAD:main
+
+      - name: Create GitHub release with PR Changelog
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)